python/lsst/ip/isr/isrMock.py

```diff
@@ -52,9 +52,19 @@ class IsrMockConfig(pexConfig.Config):
     # Detector parameters. "Exposure" parameters.
     isLsstLike = pexConfig.Field(
         dtype=bool,
-        default=True,
+        default=False,
```

Comment:
Why change the default to False here? Doesn't this mean that places that use isrMock will have to set this back to True if they wan't the behavior to be unchanged?

Reply:
This change is to match the previous behavior.  As you mentioned, the isLsstLike option was being eaten by the plateScale, so in all previous calls to IsrMockConfig, the object returned had isLsstLike=False.